fire/cli/info.py

```diff
@@ -599,7 +599,7 @@ def sag(sagsid: str, **kwargs):
         fire.cli.print(f"  Sagsid        : {sag.id}")
         fire.cli.print(f"  Oprettet      : {sag.registreringfra}")
         fire.cli.print(f"  Sagsbehandler : {sag.behandler}")
-        if sag.aktiv == True:
+        if sag.aktiv is True:
```

Comment:
Når vi nu forbedrer det kan vi lige så godt gøre det ordenligt :-)
```suggestion
        if sag.aktiv:
```